wallet/walletrpc.c

```diff
@@ -288,8 +294,36 @@ static struct command_result *json_prepare_tx(struct command *cmd,
 			   p_opt("feerate", param_feerate, &feerate_per_kw),
 			   p_opt_def("minconf", param_number, &minconf, 1),
 			   p_opt("utxos", param_utxos, &chosen_utxos),
+			   /* Note, withdraw does not have `locktime`
+			    * parameter.
+			    */
```

Comment:
Why not? Wouldn't it make sense to have most transactions as homogeneous as possible (yes, I know we can look at the output types and get an idea, but with taproot we wouldn't need to change).

Reply:
`withdraw` populates the locktime, using the same logic as bitcoind's locktime selection